src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt

```diff
@@ -80,4 +81,17 @@ class GithubWebTests : SeleniumTest()
         driver.get("http://localhost:${AppConfig()["app.port"]}/weblogin?requestedUrl=<<<<\"ksrcdg%20>>>545454")
         assertThat(driver.findElement(By.className("btn-xl")).text).isEqualTo("Log in with\nGitHub")
     }
+
+    @Test
+    fun `can trigger login with gitHub link`()
+    {
+        startApp("auth.provider=github")
+
+        driver.get(url)
+
+        val authProvider = driver.findElement(By.cssSelector(".login-link"))
```

Comment:
This is just testing that you can login, which is already tested in `can login with Github`. I just meant that you should add this line to test the actual code change like you did in the 'unit' test to check that it works through the browser. Ideally we'd be testing this with an app not at the root of the domain, which prompted the change, but I think that would be quite tricky

```suggestion
        val authProvider = driver.findElement(By.cssSelector(".login-link"))
Assertions.assertThat(authProvider.getAttribute("href").isEqualTo("${url}weblogin/external?requestedUrl=${url}")
```